thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -0,0 +1,306 @@
+<div class="manage-alert-explore">
+  {{#if (not isReplayPending)}}
+    {{!-- {{#if filteredAnomalies}} --}}
+      <div class="te-pill-selectors">
+        {{!-- Date range selector --}}
+        <div class="te-pill-selectors__label">Showing</div>
+        <ul class="te-alert-page-hzlist">
```

Comment:
Can you clarify what `hzlist` means? It's not entirely clear to me at initial glance.

Reply:
Horizontal list. Added comment in stylesheet.